### ISSUE TITLE
Travis connection to Firebase does not work.

### ISSUE THREAD

**Issue by Matthias-Schaub** [mapswipe/python-mapswipe-workers#362]
It seems like travis is not able to connect to the firebase test instance. Since unittests are run in .travis this error ocurs:

```
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.6_with_system_site_packages/lib/python3.6/site-packages/[secure]/auth.py", line 34, in firebaseDB
    firebase_admin.get_app()
  File "/home/travis/virtualenv/python3.6_with_system_site_packages/lib/python3.6/site-packages/firebase_admin/__init__.py", line 138, in get_app
    'The default Firebase app does not exist. Make sure to initialize '
ValueError: The default Firebase app does not exist. Make sure to initialize the SDK by calling initialize_app().
```

**Comment by Hagellach37** [comment]
Can you double check if this try except statements works properly?

https://github.com/mapswipe/python-mapswipe-workers/blob/45698499208611be91e8db71c2b25d6aad61c30b/mapswipe_workers/mapswipe_workers/auth.py#L31

**Comment by Hagellach37** [comment]
It also looks like that the SERVICE_ACCOUNT_KEY file is not correctly set up in travis.

```
During handling of the above exception, another exception occurred:

Traceback (most recent call last):

  File "/home/travis/build/[secure]/python-[secure]-workers/[secure]/tests/unittests/test_archive_project.py", line 11, in setUp

    self.project_id = set_up.create_test_project("build_area", results=True)

  File "/home/travis/build/[secure]/python-[secure]-workers/[secure]/tests/unittests/set_up.py", line 46, in create_test_project

    set_firebase_test_data(data_type, project_id, project_type)

  File "/home/travis/build/[secure]/python-[secure]-workers/[secure]/tests/unittests/set_up.py", line 20, in set_firebase_test_data

    fb_db = auth.firebaseDB()

  File "/home/travis/virtualenv/python3.6_with_system_site_packages/lib/python3.6/site-packages/[secure]/auth.py", line 38, in firebaseDB

    cred = credentials.Certificate(SERVICE_ACCOUNT_KEY_PATH)

  File "/home/travis/virtualenv/python3.6_with_system_site_packages/lib/python3.6/site-packages/firebase_admin/credentials.py", line 83, in __init__

    with open(cert) as json_file:

FileNotFoundError: [Errno 2] No such file or directory: '[secure]'
```

**Comment by Matthias-Schaub** [comment]
Fix in Pull Request #407
